TSOClient/FSO.Server.Database/DatabaseScripts/changes/0005_lotadmit_primarykey.sql

```diff
@@ -1,3 +1,3 @@
-﻿ALTER TABLE `fso`.`fso_lot_admit` 
```

Comment:
Must have totally blanked out when writing this. Will definitely avoid in future.